frontend/src/app/modules/workspace/personal/personal.component.html

```diff
@@ -12,10 +12,18 @@
   </div>
   <div class="header-links-right">
     <button mat-button class="button-disabled">Export Insights</button>
-    <button mat-button class="button-outlined">New Folder</button>
+    <button mat-button class="button-outlined" (click)="showNewFolderForm()" >New Folder</button>
     <button mat-button class="button-accent">New Video</button>
   </div>
 </div>
+<h2>Folders</h2>
```

Comment:
Please, use https://material.angular.io/components/dialog/overview instead of this approach

Reply:
thanks